articles/native-platforms/ios-objc/04-User-Profile.md

```diff
@@ -0,0 +1,97 @@
+---
+title: User Profile
+description: This tutorial will show you how to access the user profile from within your app, as well as how to update it.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objC',
+  pkgFilePath: 'iOS/basic-sample-objc/ObjCSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### In the Beginning
+
+#### Be familiar with Auth0
+
+This tutorial assumes you are already familiar with Auth0 and how to Sign up and Sign in using Lock or Auth0 Toolkit. **If you're not sure, check out [this tutorial](01-login.md) first.**
+
+### 1. Remember me?
+
+To get your users information, you will need the user's token. You can get it either after your sign in ([Sing in Tutorial](01-login.md)) or getting it from your Keychain ([Session Handling Tutorial](03-session-handling.md)).
```

Comment:
To get your user's information, you will need his token.
or
To get information about a user, you will need his token. (shouldn't we also be clear it should be the `idToken`, and not just any token?)